lesson_6/mosut_sorin/test.py

```diff
@@ -0,0 +1 @@
+print('Salut')
```

Comment:
Great print, but it is missing a new line at the end of file

Reply:
Beton COD

Reply:
alo?aaaaloooo? sunt eu..picasoo

Reply:
i'll fix it